_projects/2019/astropy/n-waymatch.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Developing an n-way matcher for Astropy
+desc: Develop an Astropy Table compatible n-way matcher of astronomical catalogues
+requirements:
+ - Python
+difficulty: medium
```

Comment:
is medium about right?

Reply:
I don't know what the different levels are, so I don't know how to judge that.

Reply:
fair enough. @eteq what do you think?